src/stores/media/index.ts

```diff
@@ -378,21 +382,47 @@ export const useMediaStore = defineStore('media', {
     }) {
       let errorMessage
       if (axios.isAxiosError(error)) {
-        errorMessage =
-          error.response?.status === 500
-            ? 'There was a problem with our servers'
-            : `Request failed with status ${
-                error.response?.status ?? 'unknown'
-              }`
+        // If the error is an axios error:
+        // If the error has a response property, and recieved a response that is not in the 2xx range,
+        // then the error is logged to Sentry
+        if (error.response && error.response.status >= 300) {
+          errorMessage = `Error fetching ${mediaType} from API. Request failed with status code: ${error.response.status}`
+          this.$nuxt.$sentry.captureEvent({
+            message: errorMessage,
+            extra: {
+              error,
+            },
+          })
+        } else if (!error.response && error.request) {
```

Comment:
```suggestion
        } else if (error.request) {
```
The `error.response` check is unnecessary here if the first check is simple for `if(error.response)`

Reply:
Fixed. Thanks for pointing it out!